### ISSUE TITLE
ERROR! Unexpected Exception, this is probably a bug: No module named 'psutil'

### ISSUE THREAD

**Issue by jepsenwan** [ansible/ansible-runner#705]
Hi,  I got the below error when running the playbook using python3:

ERROR! Unexpected Exception, this is probably a bug: No module named 'psutil'

But I have got all the required packages installed in my OSX env.

```
web$pip3 show ansible
Name: ansible
Version: 4.0.0
Summary: Radically simple IT automation
Home-page: https://ansible.com/
Author: Ansible, Inc.
Author-email: info@ansible.com
License: GPLv3+
Location: /Users/zj/Library/Python/3.8/lib/python/site-packages
Requires: ansible-core
Required-by: 

web$pip3 show ansible-runner
Name: ansible-runner
Version: 1.4.7
Summary: UNKNOWN
Home-page: https://github.com/ansible/ansible-runner
Author: Red Hat Ansible
Author-email: None
License: Apache
Location: /Users/zj/Library/Python/3.8/lib/python/site-packages
Requires: psutil, pexpect, six, PyYAML, python-daemon
Required-by: 

web$pip3 show psutil
Name: psutil
Version: 5.8.0
Summary: Cross-platform lib for process and system monitoring in Python.
Home-page: https://github.com/giampaolo/psutil
Author: Giampaolo Rodola
Author-email: g.rodola@gmail.com
License: BSD
Location: /Users/zj/Library/Python/3.8/lib/python/site-packages
Requires: 
Required-by: ansible-runner
```

In the artifacts, it is 250, failed, and 
```
[0;31mERROR! Unexpected Exception, this is probably a bug: No module named 'psutil'[0m
to see the full traceback, use -vvv

```

Tried downgrading the psutil to 5.7.2, but did not work. Is there something else I need to tweak? Please help out.

Thanks.

**Comment by jepsenwan** [comment]
All right fixed in #466 , fixed this by installing the fix version: ` pip3 install ansible-runner==2.0.0-alpha2`

Thanks.

**Comment by jepsenwan** [comment]
#466
